src/passes/assume.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{
+    core::{self, OrderConstraint, Expr},
+    passes::Pass
+};
+use itertools::Itertools;
+
+/// Add default assumptions to the Filament program
+pub struct Assume;
+
+impl Assume {
+    /// Iterate through the parameter constraints in a signature to see if we can add any default function assumptions
+    fn sig(sig: &core::Signature) -> Vec<core::Command> {
+        sig.param_constraints
+            .iter()
+            .filter_map(|c| match c.inner() {
+                OrderConstraint {
```

Comment:
Maybe the `FnAssume` struct should define a `matches` method or something to abstract this away? Might be useful in general.